src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
I agree that the test can be stronger. If there is a significant risk a bug slipping through and compromising the entire test suite due to the weakness of this test, then we should strengthen it. If there is no such risk, then we can simply leave it as a TODO to strengthen it later. On a related note, God Mode is a complicated yet vital piece of our infrastructure. So may be when the code is fully cleaned up we should review it on its entirety. @wkurniawan07 @thyageshm and I can do it together. I wonder how to use PRs to do a code review on an existing piece of code :-p